packages/slate-tools/tools/webpack/config/parts/core.js

```diff
@@ -104,5 +101,10 @@ module.exports = {
         to: config.get('paths.theme.dist.templates'),
       },
     ]),
+    new SlateLocalesPlugin({
+      from: config.get('paths.theme.src.sections'),
+      to: config.get('paths.theme.dist.sections'),
+      genericTemplateName: 'template.liquid',
```

Comment:
Maybe we can have this set as the default value instead of needing to declare it? It can remain an option if users want to specify another name.